keyboards/philoandgunge/nyaaaah54_rp2040/info.json

```diff
@@ -0,0 +1,111 @@
+{
+  "keyboard_name": "nyaaaah54_rp2040",
+  "manufacturer": "0xPHGU",
+  "url": "https://philoandgunge.com",
+  "maintainer": "jkaunert",
+  "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+   },
+  "matrix_pins": {
```

Comment:
Mixed indentation, should be formatted with `qmk format-json`

Reply:
all resolved. thanks for reviewing!

Reply:
Still needs formatting.

Reply:
reformatted again from terminal inside vscode, hopefully that corrects the mixed indents.